awx/main/models/jobs.py

```diff
@@ -277,6 +277,12 @@ class Meta:
         default=False,
         allows_field='credentials'
     )
+    job_shard_count = models.IntegerField(
+        blank=True,
+        default=0,
+        help_text=_("The number of jobs to split into at runtime. Will cause the Job Template to launch a workflow."),
```

Comment:
best to note in help text that 0 indicates that no workflow will be spawned, which is different than if you made it 1.